### PR TITLE
LL-2267 Images: prevent right clicks

### PR DESCRIPTION
**Pull request by LFBarreto**
prevent right click on images

This should fix the issue where users could right click image resources and save them.

### Type

Bug Fix

### Context

LL-2267

### Parts of the app affected / Test plan

Try right clicking on the logo in the password page or any logo on the manager catalog list or any baker image on a Tezos validator selection.

### CODE REVIEW

File: src/renderer/index.html [hunk 1]
```diff
@@ -1,94 +1,97 @@
...
     </div>
-  </div>
-  <div id="modals"></div>
-</body>
-
-</html>
\ No newline at end of file
+    <div id="modals"></div>
+  </body>
+</html>
```
Comment (gre) [inline src/renderer/index.html]: please undo this file change, it is not related to the commit. (if it is needed, should be another commit)

Comment (gre) [inline src/renderer/index.html]: mmh, can we avoid the big diff? edit the file without prettier.

if we want to prettier it, we should do it in another "previous" commit to make commit more atomic.

Comment (LFBarreto) [reply src/renderer/index.html]: :+1:

**Comment by gre** [comment]
squash the two commits please :)
